source/utilities.d.ts

```diff
@@ -1,3 +1,5 @@
 export type UpperCaseCharacters = 'A' | 'B' | 'C' | 'D' | 'E' | 'F' | 'G' | 'H' | 'I' | 'J' | 'K' | 'L' | 'M' | 'N' | 'O' | 'P' | 'Q' | 'R' | 'S' | 'T' | 'U' | 'V' | 'X' | 'Y' | 'Z';
 
 export type WordSeparators = '-' | '_' | ' ';
+
+export type Integers = '0' | '1' | '2' | '3' | '4' | '5' | '6' | '7' | '8' | '9';
```

Comment:
The name should probably mention that it's strings.

Reply:
Same as [here](https://github.com/sindresorhus/type-fest/pull/153/files#r531693170) – I recommend `NumericalString`

Reply:
What about `StringDigit`? Usually literal unions are singular, e.g. `type Direction = 'up' | 'down'`. If we go with that in a follow up I could also change the other type utilities in this file to be singular (reason becomes kinda clear if you try to use them in a "normal" way: `let my letter: UppercaseCharacters = 'F'`)